Source/Fuse.Controls.Panels/LayoutControl.uno

```diff
@@ -193,5 +193,14 @@ namespace Fuse.Controls
 				return LayoutDependent.Maybe;
 			}
 		}
+
+		protected override bool FastTrackDrawWithOpacity(DrawContext dc)
+		{
+			if (HasChildren) return false;
+			if (Background == null) return true;
+			
+			DrawBackground(dc, Opacity);
+			return base.FastTrackDrawWithOpacity(dc);
```

Comment:
It seems odd to return the call to `base` here yet the `(BAckground == null)` check returns `true` without checking the call to base.

Though I'm not sure this can be done correctly without a query function:

```
if (HasChildren) return false;
var bRes = base.CanFastTrackDrawWithOpacity(dc);
if (!bRes) return false;

if (Background != null) DrawBackground(dc,Opacity);
return base.FastTrackDrawWithOpacity(dc);
```